lib/services/api/settings_slicer.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  class SettingsSlicer
+    def self.slice(settings, *path)
```

Comment:
I'm open to the idea of this small service living elsewhere, but I don't think that SettingsFilterer is the right place for it since it's a service that has a single responsibility.

I personally see small classes that do one thing as a design goal, not a cost that needs to be mitigated :grin: